frontend/lib/src/components/widgets/DataFrame/columns/utils.ts

```diff
@@ -134,14 +135,15 @@ interface ErrorCell extends TextCell {
  * @return a read-only GridCell object that can be used by glide-data-grid.
  */
 export function getErrorCell(errorMsg: string, errorDetails = ""): ErrorCell {
-  errorMsg = `⚠️ ${errorMsg}`
   return {
     kind: GridCellKind.Text,
     readonly: true,
     allowOverlay: true,
-    data: errorMsg + (errorDetails ? `\n\n${errorDetails}\n` : ""),
+    data: errorMsg,
     displayData: errorMsg,
+    errorDetails: errorDetails,
     isError: true,
+    style: "faded",
   } as ErrorCell
```

Comment:
**suggestion (non-blocking):** While we're here, we should remove this unnecessary type assertion since the function return value is already typed. Keeping this type assertion may lead to the compiler skipping over issues that we actually want to be made aware of.

Reply:
Good point 👍 I removed it.